src/engine/init.c

```diff
@@ -633,7 +633,7 @@ server_init(int argc, char *argv[])
 	/* initialize the network layer */
 	ctx_nr = dss_ctx_nr_get();
 	rc = crt_init_opt(daos_sysname,
-			  CRT_FLAG_BIT_SERVER,
+			  CRT_FLAG_BIT_SERVER | CRT_FLAG_BIT_AUTO_SWIM_DISABLE,
```

Comment:
I don't think this is safe to do compared to patch provided by LiWei: https://github.com/daos-stack/daos/pull/10370 . 

This flag will disable also all alive events and might affect startup of daos when servers come alive/join.

Reply:
Right, I've tried this earlier and it definitely required more changes to bootstrap. Then, I still saw -DER_BAD_TARGETs when creating pools. Moreover, I thought we still want to see how swim responds to the workload.

Reply:
Yeah, I tried this patch and it was stuck at `dmg storage format` indefinitely. So even server start was not possible.
@liw , yeah we did wanted to check how swim responds without evictions, which we did in the run using your patch #10370 . But later it was also discussed to see how it works if  we could completely turn off swim.